app/assets/javascripts/discourse-common/addon/lib/icon-library.js

```diff
@@ -65,17 +65,15 @@ export function enableMissingIconWarning() {
   warnMissingIcons = false;
 }
 
-export function renderIcon(renderType, id, params) {
-  params ||= {};
-
+export function renderIcon(renderType, id, options = {}) {
```

Comment:
Are we 100% sure this is safe change? 😃 There might be a `null` value passed in, e.g. by some theme component?

Reply:
Reverted that diff